### ISSUE TITLE
how can I train the hrnet+ocr+segfix?

### ISSUE THREAD

**Issue by YAwei666** [openseg-group/openseg.pytorch#48]
thank you

**Comment by hsfzxjy** [comment]
@YAwei666 You may check the following instructions to reproduce the best HRNet+OCR+SegFix results on Cityscapes leaderboard:

 1. Refer to [here](https://github.com/openseg-group/openseg.pytorch/blob/master/MODEL_ZOO.md#how-to-reproduce-the-hrnet--ocr-with-mapillary-pretraining) to train an HRNet+OCR model with best performance.
 2. Apply SegFix following the instructions at [here](https://github.com/openseg-group/openseg.pytorch/blob/master/MODEL_ZOO.md#segfix-pipelines), check the "For whom want to try SegFix by using offline-generated offsets" section.

**Comment by YAwei666** [comment]
how to trian  a new segfix model on my own datasets, I have trained a HRNet+OCR, but don't know how to do next step.

**Comment by hsfzxjy** [comment]
You may check the instruction at [here](https://github.com/openseg-group/openseg.pytorch/blob/master/MODEL_ZOO.md#cityscapes-semantic-segmentation). 

For generating ground truth on your own dataset, you may create a script imitating [dt_offset_generator.py](https://github.com/openseg-group/openseg.pytorch/blob/master/lib/datasets/preprocess/cityscapes/dt_offset_generator.py), change L47 to your label mapping, and L114 to the path of your dataset. Some other minor edits might be included. Run the script and you will get the ground truth files.
